tuple.dd

```diff
@@ -3,7 +3,7 @@ Ddoc
 For documentation on tuples that are commonly present in other languages (anonymous runtime entity that groups
 different values) refer to $(LINK2 phobos/std_typecons.html#tuple, Phobos documentation on std.typecons.tuple)
 
-You may be also looking for documentation for built-in $(LINK2 ctarguments.html, compile-time argument lists)
+You may be also looking for documentation for built-in $(LINK2 ctarguments.html, compile-time sequences)
```

Comment:
While you are at it, you could update the linkt to `articles/ctarguments.html` (not required, there are redirects)

Reply:
Haven't done this, but fixed the other points. Thanks for reviewing.